hack/e2e.sh

```diff
@@ -325,6 +320,13 @@ function test_cassandracluster() {
         fail_test "Second cassandra node did not become ready"
     fi
 
+    seed_label=$(kubectl get pods --namespace "${namespace}" \
+                 cass-${CASS_NAME}-ringnodes-1 \
+                 -o jsonpath='{.metadata.labels.seed}')
```

Comment:
(may not be something that's true because of this PR, but we should always be using namespaced labels e.g. `navigator.jetstack.io/cassandra-seed=true`)

Reply:
:+1: this needs rebasing when https://github.com/jetstack/navigator/pull/270 lands, which makes the label `navigator.jetstack.io/cassandra-seed`